docs/LayoutSpecs.json

```diff
@@ -1,1365 +1,1390 @@
 {
```

Comment:
I'd suggest right before I merge your PR, manually add `sections` to the latest `LayoutSpecs.json` so that we won't be having too much conflicts moving forward.